website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+## Flipper by default
+
+[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.
+
+Flipper provides the following features out of the box:
+
+- Network Inspector
+- Layout Inspector
+- Crash Reporter
+- Device logs
+
+There is also a marketplace of custom Flipper plugins you can install:
```

Comment:
```suggestion
Flipper provides the following features out of the box:

- Reloading the app and triggering the Dev Menu.
- React DevTools
- Metro Logs
- Device Logs
- Crash Reporter
- Database Inspector
- Preference Inspector

Since Flipper is an extensible platform, there is a built-in marketplace (connected to NPM) which could feature your plugins!
```

I removed the the Network Inspector, as, IIRC, the RN network stack doesn't use the underlying libraries that the Network Inspector uses, so afaik it doesn't show anything useful normally? Or does it work in your experience?

I removed the Layout Inspector as well, it does work, but the value to RN users isn't really clear to me. If you think it is a valuable feature, feel free to put it back. 

cc @passy

Reply:
That makes sense -- I'll add the layout inspector back as this is super useful for debugging layout issues in React Native 👍

What would need to be done to get the Network plugin working?

Reply:
Afaik, but I'm on very unfamiliar territory now, it intercepts the `okhttp3` library on Android, and the native libraries on iOS. But I am not sure that RN Android uses that library at all for networking (sorry, I don't have a RN non-FB app ready at hand that does send network requests, I can gobble something together tomorrow if you want me to actually test that)

Reply:
I thought RN did use okhttp?

Reply:
It does. :) If this doesn't work it's a bug, but I haven't heard that from anyone who's used it so far.

Reply:
Just tried it, and networking plugin works on both RN and android, so we should definitely mention it :)

Reply:
Nice, thanks! I also consolidated and added descriptions to each to fill out the content and provide context where it might not be obvious.